src/OpenTelemetry.AutoInstrumentation/Configurations/TracerSettings.cs

```diff
@@ -63,7 +63,7 @@ internal class TracerSettings : Settings
     /// <summary>
     /// Gets the instrumentation options.
     /// </summary>
-    public InstrumentationOptions InstrumentationOptions { get; private set; } = new(new Configuration());
+    public InstrumentationOptions InstrumentationOptions { get; private set; } = new(new Configuration(false));
```

Comment:
Is it possible to use a named parameter consistently when creating types that take it as a parameter?

Reply:
It was the only place under src with non-named parameter. Fixed.